tests/scripts/task_python_hexagon.sh

```diff
@@ -18,10 +18,34 @@
 
 set -e
 set -u
-set -x
 
-source tests/scripts/setup-pytest-env.sh
+device_serial="simulator"
+if [ $# -ge 1 ] && [[ "$1" = "--device" ]]; then
+    shift 1
+    device_serial="$1"
+    shift
+fi
 
+source tests/scripts/setup-pytest-env.sh
 make cython3
 
+if [[ "${device_serial}" == "simulator" ]]; then
+    export TVM_TRACKER_PORT=9190
+    export TVM_TRACKER_HOST=0.0.0.0
+    env PYTHONPATH=python python3 -m tvm.exec.rpc_tracker --host "${TVM_TRACKER_HOST}" --port "${TVM_TRACKER_PORT}" &
```

Comment:
I’m not good enough at bash to know off the top of my head, but does this Python process exit when the script is done even if there is an error before the `kill` below?

Reply:
probably not, so I guess whether the script reaches the `kill` command depends on the brhaviour of `run_pytest` command in case an error happens on one of the tests.
I think there are other ways to run tracker in python test files which we can replace them.

Reply:
The bash way to ensure it gets cleaned up would be to use `trap my_cleanup_function EXIT` to run a function when the script exits.  That said, I like to idea of having it all in python, for uniformity.

Reply:
This all runs in docker in CI, so from that perspective it doesn't matter if the script leaves some processes behind.

We were hitting this issue in our downstream CI and we start the tracker in a separate process group, then terminate all processes from the group with a single `kill -TERM -${PGID}` command.  I don't know what your approach to running tracker from python would be, but keep in mind that python is not very good at guaranteeing global cleanup: if the program exits in some abnormal way (say, due to a syntax error), the execution will just stop and no "destructors" will be run.  

In any case, this issue was present even before this refactoring, so this is not a regression.  I'm ok with checking this in, and addressing the cleanup issue in another PR.

Reply:
Thanks for the feedbacks. We could fix this in a follow up PR.